_analyzers/tokenizers/edge-n-gram.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Edge n-gram
+parent: Tokenizers
+nav_order: 40
+---
+
+# Edge n-gram tokenizer
+
+The `edge_ngram` tokenizer generates partial word tokens, or n-grams, starting from the beginning of each word. It splits the text based on specified characters and produces tokens with lengths defined by a minimum and maximum length. This tokenizer is particularly useful for implementing search-as-you-type functionality. 
+
+Edge n-grams are ideal for autocomplete searches where the order of the words may vary, such as with product names or addresses. For more information, see [Autocomplete]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/autocomplete/). However, for text with a fixed order, like movie or song titles, the completion suggester may be more efficient.
+
+By default, the `edge n-gram` tokenizer produces tokens with a minimum length of `1` and a maximum length of `2`. For example, analyzing the text `OpenSearch` with the default configuration will produce the `O` and `Op` n-grams. These short n-grams are often not sufficient for meaningful searches, so configuring the tokenizer is necessary to adjust the gram lengths.
+
```

Comment:
Line 14: "not sufficient for meaningful searches" is a bit too ambiguous/imprecise. Please clarify/simplify.